File: 02-EstructurasDeControl/programasPython/expresiones.py

"""Declaracion de variables
En esta parte del codigo se van
 a declarar varios tipos de variables"""

 #Expresiones aritmeticas
suma = 3 + 5
resta = 10 -12
multi = 24 * 23
div = 10 / 2
divEntera = 12// 5 #Division entera
mod = 10 % 2
potencia = 4**2 #Potencia

salida = f"Hola soy la suma: str{suma}"
print(salida)
salida = 'Hola soy la suma' 
print(salida,suma)
salida = 'Hola soy la suma' + str(suma)
print(salida)
salida = 'Hola soy la suma' +str(suma)
print(salida)
#Interpolacion de strings
salida = f"""
==== Expresones Aritmeticas ====
La suma es: {suma}
La resta es: {resta}
La multiplicacion es: {multi}
La division es: {div}
La division entera es: {divEntera}
El modulo es: {mod}
La potencia es: {potencia}
"""
print(salida)

File: 02-EstructurasDeControl/programasJava/Expresiones.java

import javax.swing.JOptionPane;

public class Expresiones {
    public static void main(String[] args) {
        //Declaracion de variables
        /*
        *En esta parte del codigo se van
        * a declarar varios tipos de variables
         */
        int suma;
        int resta;
        int multi, div, mod, expresion, area;
        double resultado = 0.0, potencia;
        boolean valor = true, valor2 = false;
        float f1 = (float)8.9, f2 = 10.3f;
        double d1 = 12;

        //Definir una constante
        final double PI = 3.1416;
        String salida = "";

        //Expresiones aritmeticas
        suma = 3 + 5;
        resta = 10 -12;
        multi = 24 * 23;
        div = 10 / 2;
        mod = 10 % 2;
        potencia = Math.pow(4,2);

        salida = "===== Expresiones Aritmeticos =====" +
        "\nLa suma es: "+ suma +
                "\nLa resta es: "+ resta +
                "\nLa multiplicacion es: "+ multi +
                "\nLa division es: "+ div +
                "\nLa modulo es: "+ mod +
                "\nLa potencia es: "+ potencia
        ;

        JOptionPane.showMessageDialog(null, salida);


    }
}